forge/db/controllers/ProjectTemplate.js

```diff
@@ -224,11 +224,17 @@ module.exports = {
    * @param {*} settings the new settings to merge in
    * @param {*} options options for the merge
    * @param {boolean} options.mergeEnvVars if true, merge the env vars (new keys added, existing keys untouched, removed keys untouched)
+   * @param {boolean} options.updateEditorSettings if true will override the current editor settings for certain keys
```

Comment:
```suggestion
   * @param {boolean} options.mergeEditorSettings if false (default), will overwrite all settings. If true, it will merge the settings together - only overwriting certain ones.
```